rcongui/src/components/useStyles.js

```diff
@@ -64,10 +64,77 @@ const useStyles = makeStyles((theme) => {
       textAlign: "left",
       padding: theme.spacing(2),
     },
+    // from discord_chat.py...
+    // RED = 0xA62019
+    // LIGHT_RED = 0xF93A2F
+    // BLUE = 0x006798
+    // LIGHT_BLUE = 0x0099E1
+    // GREEN = 0x07DA63
     logs: {
       margin: 0,
       whiteSpace: "pre-wrap",
     },
+    logsChatAxis: {
+      color: "#F93A2F",
```

Comment:
these colors work well with the light theme. Did you check all other themes as well? Would probably make sense to use a different color palette per theme. What do you thinnk?

Reply:
They aren't great with dark mode, from what I recall.

I don't know how to correctly interact with the theming engine to switch between colours for dark vs. light mode.

I put the switch on the logs page to make it a bit obvious on how to turn it off if the colours weren't usable.

Reply:
Actually the colors are better than I remembered in the dark theme. A slight tweak to make the blue lighter and it looks pretty good on both.
![Screenshot 2024-02-14 085032](https://github.com/MarechJ/hll_rcon_tool/assets/2712638/c092ca5c-61a3-457e-ac63-33fb359d8ecd)
![Screenshot 2024-02-14 085008](https://github.com/MarechJ/hll_rcon_tool/assets/2712638/4051af4e-8a9c-47be-a5f5-e780088fa986)

Hopefully I'll have time to merge the changes into the PR branch tonight.

Reply:
ok, that looks better than I expected as well 👍